example/feature_demo/demo_service.proto

```diff
@@ -38,25 +49,38 @@ message UpdateIntPointResponse {
 }
 
 message DeleteIntPointRequest {
-        uint32 id = 1;
+    // Only the id is needed for a delete request
+    uint32 id = 1;
 }
 
+// By convention, on DELETE no response is expected
```

Comment:
can we still include an empty `DeleteIntPointResponse` message? I'd prefer to set that as the pattern than an empty response